common-setup.sh

```diff
@@ -133,17 +141,17 @@ EOF
     fi
 }
 
-allow_broken_macos="$HOME/.local/share/reflex-platform/allow-broken-macos"
-if (sw_vers | grep "ProductVersion" | grep "\(10.13.0\|10.13.2\)$" || false)  &> /dev/null; then
```

Comment:
Oops, this should have been 10.13.0 and 10.13.1.

Reply:
I just reverted to that fix instead. The version comparison was cool and all but just too much for this after all. Thanks for clarifying.

Reply:
Neat - I'll try to test this in a couple hours